pkg/agent/controller/networkpolicy/reject.go

```diff
@@ -115,8 +117,22 @@ func (c *Controller) rejectRequest(pktIn *ofctrl.PacketIn) error {
 		isIPv6 = true
 	}
 
-	sIface, srcFound := c.ifaceStore.GetInterfaceByIP(srcIP)
-	dIface, dstFound := c.ifaceStore.GetInterfaceByIP(dstIP)
+	sIface, srcIsLocal := c.ifaceStore.GetInterfaceByIP(srcIP)
+	dIface, dstIsLocal := c.ifaceStore.GetInterfaceByIP(dstIP)
+	dstIsDirect := dstIsLocal
```

Comment:
Better to add a comment to explain what does "dstIsDirect" mean, like "The reject packet destination is on the same Node and the reject packet can be forwarded without leaving the OVS bridge".